### ISSUE TITLE
Crash on the care card

### ISSUE THREAD

**Issue by mihaelamiches** [carekit-apple/CareKit#84]
When tapping an intervention row on the Care Card with the accessibility inspector enabled, a exception is thrown:
```Swift 
2016-07-07 12:53:21.934 OCKSample[31203:5647838] -[UITableViewCellAccessibilityElement interventionEvents]: unrecognized selector sent to instance 0x7f9471c731e0
2016-07-07 12:53:21.936 OCKSample[31203:5647838] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[UITableViewCellAccessibilityElement interventionEvents]: unrecognized selector sent to instance 0x7f9471c731e0'
*** First throw call stack:
(
	0   CoreFoundation                      0x0000000102cc0d85 __exceptionPreprocess + 165
	1   libobjc.A.dylib                     0x0000000104b92deb objc_exception_throw + 48
	2   CoreFoundation                      0x0000000102cc9d3d -[NSObject(NSObject) doesNotRecognizeSelector:] + 205
	3   CoreFoundation                      0x0000000102c0fcfa ___forwarding___ + 970
	4   CoreFoundation                      0x0000000102c0f8a8 _CF_forwarding_prep_0 + 120
	5   CareKit                             0x00000001029a9a72 -[CareCardAccessibilityElement accessibilityValue] + 130
	6   UIAccessibility                     0x00000001162d0894 -[NSObject(AXPrivCategory) accessibilityAttributeValue:] + 1449
	7   UIAccessibility                     0x00000001162bc449 _copyMultipleAttributeValuesCallback + 381
	8   AXRuntime                           0x0000000116350dea _AXXMIGCopyMultipleAttributeValues + 215
	9   AXRuntime                           0x000000011634bc98 _XCopyMultipleAttributeValues + 467
	10  AXRuntime                           0x0000000116358e8f mshMIGPerform + 235
	11  CoreFoundation                      0x0000000102be6579 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE1_PERFORM_FUNCTION__ + 41
	12  CoreFoundation                      0x0000000102be64e9 __CFRunLoopDoSource1 + 473
	13  CoreFoundation                      0x0000000102bdbc60 __CFRunLoopRun + 2272
	14  CoreFoundation                      0x0000000102bdb0f8 CFRunLoopRunSpecific + 488
	15  GraphicsServices                    0x00000001070ecad2 GSEventRunModal + 161
	16  UIKit                               0x0000000103616f09 UIApplicationMain + 171
	17  OCKSample                           0x0000000102501be2 main + 114
	18  libdyld.dylib                       0x0000000107c7292d start + 1
)
libc++abi.dylib: terminating with uncaught exception of type NSException
````

Implicitly, it also affects any UITests that involve the OCKCareCardViewController

**Comment by YuanZhu-apple** [comment]
@mihaelamiches Which branch did you use? 
It seems I cannot repro it on master branch.

**Comment by mihaelamiches** [comment]
@YuanZhu-apple thanks for looking into this. With the master branch up to date, I am able to replicate the issue only the 5s and 6s Plus simulators. All good on the 6, 6 Plus, 6s

**Comment by YuanZhu-apple** [comment]
@umerkhan-apple Assigned to you:)

**Comment by mihaelamiches** [comment]
I looked a bit closer into the issue, and I was able to replicate each time when I run a UITest that interacts with the care card. 

When tapping on a intervention event button or on the intervention row, a wild crash occurs when setting the accessibility value for the OCKCareCardTableViewCell. The issue there seems to be that the CareCardAccessibilityElement is not a OCKCareCardTableViewCell thus not responding to the interventionEvens selector. 
This only happens to me now with ui tests. When I run now the accessibility inspector over the elements, the accessibility value is set correctly. 

I will open a pull request shortly, that fixes the problem I'm seeing here locally. It's a bit of a weird situation since I can't explain why the problem only arises when ui testing 
![b7e](https://cloud.githubusercontent.com/assets/9751726/17623054/0c196604-6096-11e6-91d7-19adda586734.jpg)
